_data/link.yml

```diff
@@ -55,30 +55,46 @@ Type:
 
 # Channel Definitions
 ConditionalStringLegendDef:
-  name: FieldDef
-  link: "encoding.html#def"
+  name: MarkPropDef
+  link: "encoding.html#mark-prop"
 ConditionalNumberLegendDef:
-  name: FieldDef
-  link: "encoding.html#def"
+  name: MarkPropDef
+  link: "encoding.html#mark-prop"
 ConditionalTextDef:
-  name: FieldDef
-  link: "encoding.html#def"
+  name: TextDef
+  link: "encoding.html#text"
 FieldDef:
-  link: "encoding.html#def"
+  link: "encoding.html#field"
 FacetFieldDef:
   link: "facet.html#facet-field-definition"
 OrderFieldDef:
-  name: FieldDef
-  link: "encoding.html#def"
+  link: "encoding.html#order"
 PositionFieldDef:
-  name: FieldDef
-  link: "encoding.html#def"
+  link: "encoding.html#position-field-def"
 NumberValueDef:
   name: ValueDef
   link: "encoding.html#value"
 
-# Scale / Axis / Legend
+# FieldDefinition
+Aggregate:
+  link: "aggregate.html"
+Axis:
+  link: axis.html
+Field:
+  link: field.html
+Legend:
+  link: legend.html
+Scale:
+  link: scale.html
+SortOrder:
```

Comment:
After applying `@nullable` to Sort, then it appears like this 
![image](https://user-images.githubusercontent.com/111269/29244707-c9732ab2-7f73-11e7-9745-cbfba3f39850.png)

because our Jekyll engine doesn't handle `(SortOrder | SortField) |` Null well